modules/administration_manual/pages/enterprise/classification_and_policy_enforcement.adoc

```diff
@@ -0,0 +1,366 @@
+= Document Classification and Policy Enforcement
+
+When dealing with large amounts of data in an enterprise, it is essential to have mechanisms in place that allow you to stay in control of data flows.
+To implement such mechanisms the first step to take is to define guidelines that describe how the content of different security levels have to be treated.
+
+Depending on the industry, such information security guidelines can originate from regulatory requirements, from recommendations of industry associations, or they can be self-imposed if there's no external factor but internal risk management requirements that demand special treatment for specific information.
+
+The leading information security standard https://www.iso.org/isoiec-27001-information-security.html[ISO 27001] defines guidelines for managing information security which can be certified.
+More specifically:
+
+. Information should enter an asset inventory (A.8.1.1)
+. Information should be classified (A.8.2.1)
+. Information should be labeled (A.8.2.2)
+. Information should be handled in a secure way (A.8.2.3)
+
+As the leading international standard and certification for information security, ISO 27001 https://www.certificationeurope.com/app/uploads/2018/05/GDPR-ISO-27001-Mapping-Guide.pdf[covers 75-80% of the GDPR]. 
+This makes it the ideal framework choice to support https://gdpr-info.eu[GDPR] compliance requirements.
+
```

Comment:
I would add here the following as this table is really useful:

Please see the following link as an example to match the mentioned ISO Controls to the relevant General Data Protection Regulation (GDPR) Articles (https://www.certificationeurope.com/app/uploads/2018/05/GDPR-ISO-27001-Mapping-Guide.pdf)

Add the link beautyfied as usual.

Reply:
Change added.